lib/brain.js

```diff
@@ -1784,10 +1793,73 @@ Brain.prototype._start = function _start () {
     return this.smsCompliance()
   }
 
+  const printPaperWallet = deviceConfig.compliance.paperWallet &&
+                           deviceConfig.kioskPrinter &&
+                           this.tx.cryptoCode === 'BTC'
```

Comment:
> What happens if the coin isn't BTC?

@joshmh what should happen if the coin isn't BTC? Do we need a screen for that 'misconfiguration'?

Reply:
The operator shouldn't allow unsupported currencies, so no need for this. We can just send the user back to idle.